src/transport/SessionManager.h

```diff
@@ -174,8 +174,155 @@ class DLL_EXPORT SessionManager : public TransportMgrDelegate, public FabricTabl
     Optional<SessionHandle> AllocateSession(Transport::SecureSession::Type secureSessionType,
                                             const ScopedNodeId & sessionEvictionHint);
 
+    /**
+     * @brief
+     *  A set of templated helper function that call a provided lambda
+     *  on all sessions in the underlying session table that match the provided
+     *  query criteria.
+     *
+     */
+
+    /**
+     * @brief
+     *
+     * Call the provided lambda on sessions that match the provided ScopedNodeId.
```

Comment:
The _remote_ side matches the provided ScopedNodeId, right?  That should be clearer in the comment.